src/clj/game/core/hosting.clj

```diff
@@ -71,7 +71,7 @@
 
        ;; events should be registered for: runner cards that are installed; corp cards that are Operations, or are installed and rezzed
        (when (or (is-type? target "Operation")
-                 (is-type? target "Event")
+                 (and (is-type? target "Event") (false? facedown))
```

Comment:
Use `(not facedown)`, which allows `facedown` to be `nil`. (`(false? nil)` returns false.)